### PR TITLE
Add note to convimg doc.

### PR DESCRIPTION
**Pull request by felicianotech**
Added a note on what image tags we won't build.

If anyone has a suggestion for a better heading though, feel free to change it.

### CODE REVIEW

File: jekyll/_cci2/circleci-images.md [hunk 1]
```diff
@@ -252,6 +252,14 @@ The following packages are installed via `curl` or other means.
 - [dockerize](https://github.com/jwilder/dockerize)
 - [jq](https://stedolan.github.io/jq/)
 
+
+## What Isn't Available
```
Comment (rosieyohannan) [inline jekyll/_cci2/circleci-images.md]:
```suggestion
## Out of Scope
```

File: jekyll/_cci2/circleci-images.md [hunk 2]
```diff
@@ -252,6 +252,14 @@ The following packages are installed via `curl` or other means.
...
+
+1. If an image isn't listed above, it isn't available. As the Convenience Image program is revamped, proposals for new images aren't currently being accepted.
```
Comment (rosieyohannan) [inline jekyll/_cci2/circleci-images.md]:
```suggestion
1. If an image isn't listed above, it is not available. As the Convenience Image program is revamped, proposals for new images are not currently being accepted.
```

File: jekyll/_cci2/circleci-images.md [hunk 3]
```diff
@@ -252,6 +252,14 @@ The following packages are installed via `curl` or other means.
...
+1. Old versions of software won't be rebuilt. Once an upstream image stops building the tag for a specific release, say Node.js v8.1.0, then we stop building it too. This means other tools in that image, such as `npm` in this example, will no longer be updated either.
```
Comment (rosieyohannan) [inline jekyll/_cci2/circleci-images.md]:
```suggestion
1. Old versions of software will not be rebuilt. Once an upstream image stops building the tag for a specific release, say Node.js v8.1.0, then we stop building it too. This means other tools in that image, such as `npm` in this example, will no longer be updated either.
```

File: jekyll/_cci2/circleci-images.md [hunk 4]
```diff
@@ -252,6 +252,14 @@ The following packages are installed via `curl` or other means.
...
+1. We don't support building preview, beta, or release candidate images tags. On occasion they'll be available but these tags tend to cause our build system for Convenience Images to fail. If you need a non-stable release of of language, we suggest installing it via [an orb](https://circleci.com/orbs/) or a custom Docker image instead.
```
Comment (rosieyohannan) [inline jekyll/_cci2/circleci-images.md]:
```suggestion
1. We don't support building preview, beta, or release candidate images tags. On occasion they'll be available but these tags tend to cause our build system for Convenience Images to fail. If you need a non-stable release or language, we suggest installing it via [an orb](https://circleci.com/orbs/) or a custom Docker image instead.
```

Comment (rosieyohannan) [reply jekyll/_cci2/circleci-images.md]: Not sure this is what was meant? Just took a guess :-)

File: jekyll/_cci2/circleci-images.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/circleci-images.md]: Maybe? Not sure this is quite right...

Comment (felicianotech) [reply jekyll/_cci2/circleci-images.md]: Umm. I'm still not sure but I think it's better than mine.

File: jekyll/_cci2/circleci-images.md (hunk 4 above)
Comment (felicianotech) [reply jekyll/_cci2/circleci-images.md]: This should be "of a". I'll push a commit.

**Comment by felicianotech** [comment]
@rosieyohannan I added all your suggestions but one. For that one I submitted a new commit with a fix.

**Pull request merged**